README.md

```diff
@@ -1,70 +1,61 @@
 # mini-lab
 
-The mini-lab is a small, virtual setup to locally run the metal-stack. It deploys the metal control plane and a partition with two simulated leaf switches. The lab can be used for trying out metal-stack, demonstration purposes or development.
+The mini-lab is a small, virtual setup to locally run the metal-stack. It deploys the metal control plane and a metal-stack partition with two simulated leaf switches. The lab can be used for trying out metal-stack, demonstration purposes or development.
 
 ![overview components](docs/overview.png)
```

Comment:
Should we redraw it completely, similar to how `containerlab` presents topologies in it's docs: https://containerlab.srlinux.dev/lab-examples/cfg-clos/#description, but with different(metal-stack like) palette?

Reply:
That'd be great. But it's also quite a lot of work. :sweat:

Reply:
I will try to do some proto. But there's big chance it will look quirky)